wof/wofpy_config.py

```diff
@@ -25,11 +28,13 @@
 
 Options:
   -h --help     Show this screen.
-  -v --version     Show version.
+  -v --version  Show version.
+  --mode=test   deploy mode [default: test]
 """
 
-_ROOT = os.path.abspath(os.path.join(os.pardir, os.path.dirname(__file__)))
-_CONFIG = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
+_ROOT = os.path.abspath(os.path.dirname(wof.__file__))
+_CONFIG = os.path.join(_ROOT, 'examples', 'production_configs')
+_ODM2_TIMESERIES = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
```

Comment:
I also prefer `odm2/timeseries`. I was just following https://github.com/ODM2/WOFpy/issues/132#issuecomment-304358051

Please let me know that is best for you and I'll modify.